rewrite-java-core/pom.xml

```diff
@@ -95,6 +95,26 @@
             <artifactId>jackson-dataformat-smile</artifactId>
             <version>2.15.0</version>
         </dependency>
-
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.34</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
```

Comment:
why do we need to add this dependency here?

Reply:
Removed unnecessary Lombok dependency in commit [9364625](https://github.com/Azure/azsdk-java-rewrite-recipes/pull/16/commits/936462508c26abd77a1ec4cea422690d433768a2)

Reply:
Added back the lombok dependency as it was required for the new try-catch recipe